_includes/specials/taproot/en/12-backups.md

```diff
@@ -0,0 +1,101 @@
+    you can spend using just one after six months.
+
+    In the normal case of using all three devices, your onchain script
+    is maximally efficient and private.  In the other cases, it's a bit
+    less efficient but may still be reasonably private (your script and
+    its tree depth will look similar to the scripts and depths used in
+    many other contracts).
+
+- **Social recovery for backups and security:** the example above is
+  great at protecting you if one of your devices gets stolen by an
+  attacker, but what happens if two of your devices are stolen?  Also,
+  if you frequently use your wallet, do you really want to wait even a
+  month before you can start spending again after losing a device?
+
+    Taproot makes it easy, cheap, and private to add a social element to
+    your backups.  In addition to the scripts in the previous example,
+    you can also allow immediate spending of your bitcoins with two of
+    your devices plus signatures from two of your friends or family
+    members.  Or immediate spending with only a single one of your keys
+    plus signatures from five people you trust.  (A similar non-social
+    version of this would simply be using extra devices or seed phrases
+    you have stored in secure locations.)
+
+- **Combining time and social thresholds for inheritance:** combining
+  the techniques above, you can allow someone or a group of people to
+  recover your funds in case you suddenly die or become incapacitated.
+  For example, if you haven't moved your bitcoins for six months, you
+  can allow your lawyer and any three of your five most trusted
+  relatives to spend your coins.  If you normally move your bitcoins
+  every six months anyway, this inheritance preparation has no added
+  onchain cost for as long as you live and is completely private from
+  outside observers.  You can even keep the transactions you make
+  private from your lawyer and family as long as you have a reliable way
+  for them to learn your wallet's extended public key (xpub) after your
+  death.
+
+    Please note that making it possible for your heirs' to spend your
+    bitcoins doesn't mean that they'll be able to spend those coins
+    legally.  We recommend that anyone planning to pass on Bitcoins read
+    *Cryptoasset Inheritance Planning* by Pamela Morgan ([physical book
+    and DRM'd ebook][cip amazon] or [DRM-free ebook][cip aantonop]) and
+    use its information to discuss details with a local expert in estate
+    planning.
+
+- **Compromise detection:** an idea [proposed][tree signatures] prior to
+  the invention of taproot is to put a key controlling some amount of
+  bitcoin on all of the devices you care about as a way of detecting
+  when the device has been compromised.  If the amount of bitcoin is
+  large enough, the attacker will probably spend it to themselves for
+  the immediate gain rather than wait to use their illicit access in a
```

Comment:
```suggestion
  immediate gain rather than wait to use their illicit access in a
```

Reply:
I think "the immediate gain" is ok here.